From dd014eb3277588378b196ed1fc14633a2fe6c17e Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jpetridis@gnome.org>
Date: Mon, 18 Aug 2025 01:06:36 +0100
Subject: [PATCH 1/2] project: Specify more limits to the CAS configs

These are disabled by default due to backward compatibility
so we need to explicitly set some values for timeouts and
retry policies.

This also bumps the min-version to 2.5 which is when these
options were introduced.
---
 project.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/project.conf b/project.conf
index 257d1c38b..bc06001ed 100644
--- a/project.conf
+++ b/project.conf
@@ -146,11 +146,17 @@ artifacts:
 - url: https://cache.freedesktop-sdk.io:11001
   connection-config:
     keepalive-time: 60
+    retry-limit: 5
+    retry-delay: 500
+    request-timeout: 180
 
 source-caches:
 - url: https://cache.freedesktop-sdk.io:11001
   connection-config:
     keepalive-time: 60
+    retry-limit: 5
+    retry-delay: 500
+    request-timeout: 180
 
 sandbox:
   build-arch: '%{target_arch}'
-- 
2.55.0

